spec/helpers/structured_data_helper_spec.rb

```diff
@@ -104,39 +104,26 @@
     end
   end
 
-  describe ".home_structured_data" do
+  describe ".government_organization_structured_data" do
     subject(:data) { JSON.parse(script_tag.content, symbolize_names: true) }
 
-    let(:html) { home_structured_data }
+    let(:html) { government_organization_structured_data }
     let(:script_tag) { Nokogiri::HTML.parse(html).at_css("script") }
 
-    before { enable_structured_data(:web_site) }
+    before { enable_structured_data(:government_organization) }
 
     it "returns nil when disabled by config" do
-      disable_structured_data(:web_site)
+      disable_structured_data(:government_organization)
       expect(script_tag).to be_nil
     end
 
     it "includes site information" do
       expect(data).to include({
-        "@type": "WebSite",
```

Comment:
I'm presuming that we can only define one `@type` and that is why we are removing this @sarahcrack ?